tests/API/Management/ConnectionsTestMocked.php

```diff
@@ -0,0 +1,342 @@
+        $fields         = null;
+        $include_fields = null;
+        $page           = 0;
+        $per_page       = 5;
+        $api->call()->connections->getAll($strategy, $fields, $include_fields, $page, $per_page);
+
+        $this->assertEquals( 'GET', $api->getHistoryMethod() );
+        $this->assertStringStartsWith( 'https://api.test.local/api/v2/connections', $api->getHistoryUrl() );
+        $this->assertContains( 'page=0', $api->getHistoryQuery() );
+        $this->assertContains( 'per_page=5', $api->getHistoryQuery() );
+    }
+
+    /**
+     * Test a getAll request with additional parameters added.
+     *
+     * @return void
+     *
+     * @throws \Exception Unsuccessful HTTP call.
+     */
+    public function testThatConnectionsGetAllAddsExtraParams()
+    {
+        $api = new MockApi( [ new Response( 200, self::$headers ) ] );
+
+        $strategy       = null;
+        $fields         = null;
+        $include_fields = null;
+        $page           = null;
+        $per_page       = null;
+        $add_params     = ['param1' => 'value1', 'param2' => 'value2'];
+        $api->call()->connections->getAll($strategy, $fields, $include_fields, $page, $per_page, $add_params);
+
+        $this->assertEquals( 'GET', $api->getHistoryMethod() );
+        $this->assertStringStartsWith( 'https://api.test.local/api/v2/connections', $api->getHistoryUrl() );
+        $this->assertContains( 'param1=value1', $api->getHistoryQuery() );
+        $this->assertContains( 'param2=value2', $api->getHistoryQuery() );
+    }
+
+    /**
+     * Test a basic get request.
+     *
+     * @return void
+     *
+     * @throws \Exception Unsuccessful HTTP call.
+     */
+    public function testConnectionsGet()
+    {
+        $api = new MockApi( [ new Response( 200, self::$headers ) ] );
+
+        $id = 'con_testConnection10';
+        $api->call()->connections->get($id);
+
+        $this->assertEquals( 'GET', $api->getHistoryMethod() );
+        $this->assertEquals( 'https://api.test.local/api/v2/connections/'.$id, $api->getHistoryUrl() );
+        $this->assertEmpty( $api->getHistoryQuery() );
+    }
+
+    /**
+     * Test a get call with included fields.
+     *
+     * @return void
+     *
+     * @throws \Exception Unsuccessful HTTP call.
+     */
+    public function testThatConnectionsGetIncludesFields()
+    {
+        $api = new MockApi( [ new Response( 200, self::$headers ), new Response( 200, self::$headers ) ] );
+
+        $id     = 'con_testConnection10';
+        $fields = ['name', 'strategy'];
+        $api->call()->connections->get($id, $fields);
+
+        $this->assertEquals( 'GET', $api->getHistoryMethod() );
+        $this->assertStringStartsWith( 'https://api.test.local/api/v2/connections/'.$id, $api->getHistoryUrl() );
+        $this->assertContains( 'fields='.implode(',', $fields), $api->getHistoryQuery() );
+        $this->assertNotContains( 'include_fields=true', $api->getHistoryQuery() );
```

Comment:
This test is supposed to assert that by default this property is not sent, correct? I would check against `include_fields` (without any value) so that if the property is set to false this would fail as well.